### PR TITLE
Add Counterstrike mission Siberian Conflict 3: Wasteland

### PR DESCRIPTION
**Pull request by Smittytron**
This is an incredibly straight forward kill the enemy mission.

### CODE REVIEW

File: mods/ra/maps/siberian-conflict-3-wasteland/rules.yaml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+	PlayerResources:
+		DefaultCash: 5200
+
+MONEYCRATE:
+	GiveCashCrateAction:
+		Amount: 2000
+		SelectionShares: 1
+		UseCashTick: true
+
+PARATROOPERS:
```
Comment (Mailaender) [inline mods/ra/maps/siberian-conflict-3-wasteland/rules.yaml:27]:
```suggestion
powerproxy.paratroopers::
```

or fix
> Error: Actor type `paratroopers` does not define a default visibility type!

Comment (Smittytron) [reply mods/ra/maps/siberian-conflict-3-wasteland/rules.yaml:27]: Updated

**Comment by KautionORA** [comment]
This is indeed pretty straight forward, but it plays  really well, having 2 mcvs early on is weird but definetly fun for expanding and taking all the ore (there is lots of eco :P)  played on hard, a little challenging but difficulty seems fine.

**Comment by darcyrush** [comment]
Mission plays well on hard

**Review by Mailaender — APPROVED** [review]

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/cabfa1891b3ac9652ddfc476d9680c47ff176008...12f39251eb9d335f5a9f934d12e461a7061c649d

**Pull request merged**